hooks/__tests__/useMediaQuery.test.tsx

```diff
@@ -41,7 +42,7 @@ describe('useMediaQuery', () => {
   it('should subscribe for media changes', () => {
     const listenerMock = jest
       .fn()
-      .mockImplementation((_: any, handler: Function) => {
+      .mockImplementation((_: unknown, handler: MediaQueryHandler) => {
```

Comment:
`Function` is a no-no, so I exported a type which was inspired from the implementation anyways, a noop returning void